docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,41 @@
+# Plan IAM integration strategy
```

Comment:
You'll need to add the front matter to this Markdown file. See guidelines here: https://grafana.com/docs/writers-toolkit/writing-guide/front-matter/